content/blog/2021/01/DebugInVSCodeWithCamelK1.3.0/index.md

```diff
@@ -0,0 +1,93 @@
+---
+title: "How to Java debug a Camel K integration in VS Code with Camel K 1.3.0"
+date: 2021-01-20
+authors: [apupier]
+categories: ["Tooling","Camel-k"]
+preview: "How to Java debug a Camel K integration in VS Code with Camel K 1.3.0"
+---
+
+Camel K 1.3.0 is providing a new feature to Java debug deployed integrations. Even if Camel K 1.3.0 is [not fully compatible](https://github.com/apache/camel-k/issues/1872) with VS Code Tooling for Apache Camel K extension, it remains possible to leverage the VS Code Java debugging capabilities.
+
+I recorded a [video](https://youtu.be/cFPuxd1Je3c) which is following the steps in this blogpost. I let you choose you preferred way to discover how to java debug your Camel K integration in VS Code.
+
+# Requirements
+
+- [VS Code Extension Pack for Apache Camel](https://marketplace.visualstudio.com/items?itemName=redhat.apache-camel-extension-pack)
+- Uncheck _Camelk -> Integrations: Auto-upgrade_ in _File -> Preferences -> Settings -> Extensions -> Apache Camel K Tooling_
+- [Camel K 1.3.0 binary](https://camel.apache.org/download/#camel-k) available on command-line
```

Comment:
this needs to be fixed before https://github.com/apache/camel-k/issues/1867#issuecomment-762946290

Reply:
PR provided https://github.com/apache/camel-website/pull/530

Reply:
merged